dm/config/subtask.go

```diff
@@ -223,27 +223,27 @@ func (c *SubTaskConfig) Toml() (string, error) {
 }
 
 // DecodeFile loads and decodes config from file
-func (c *SubTaskConfig) DecodeFile(fpath string) error {
+func (c *SubTaskConfig) DecodeFile(fpath string, decryptPassword bool) error {
```

Comment:
How about choosing a better variable name rather than `decryptPassword`, this may be misleading it will update this `SubTaskConfig` with the decrypted password?

Reply:
update name to `verifyDecryptPassword` in edca6b4